pandas/tests/series/methods/test_clip.py

```diff
@@ -40,6 +40,21 @@ def test_clip_types_and_nulls(self):
             assert list(isna(s)) == list(isna(lower))
             assert list(isna(s)) == list(isna(upper))
 
+    @pytest.mark.parametrize("dtypes", ["Float64", "Int64", "Float32", "Int32"])
```

Comment:
we have fixtures for all of these look in pandas/conftest.py

Reply:
I am not familiar with testing but I think I got it this time. I also expanded the test to cover all nulls as there was a fixture for it already. Thanks for your tips!